src/components/invoice/EditInvoiceViewModel.js

```diff
@@ -34,7 +34,7 @@ export default class EditInvoiceViewModel {
     this.attachments = obj.attachments || [{type: 'pdf'}];
     this.extraFields = obj.extraFields || [];
     this.isQuotation = obj.isQuotation || false;
-    this.createdOn = new Date().getTime();
+    this.createdOn = obj.createdOn || moment();
```

Comment:
Bug? Indien de createdOn leeg is (=het geval voor alle bestaande invoices) dan zal de createdOn gezet worden op vandaag bij de eerste update?